source/shared/cpp/ObjectModel/HostConfig.h

```diff
@@ -12,6 +12,7 @@ struct SpacingDefinition
     unsigned int right = 0;
     unsigned int top = 0;
     unsigned int bottom = 0;
+    SpacingDefinition(int left = 0, int right = 0, int top = 0, int bottom = 0):left(left), right(right), top(top), bottom(bottom){};
```

Comment:
No bad side effects from moving to C++14, assuming this is a private header.

---
In reply to: [122286906](https://github.com/Microsoft/AdaptiveCards/pull/423#discussion_r122286906) [](ancestors = 122286906,122282237)